libs/designsystem/src/lib/components/item-group/list-experimental.component.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { IonItem } from '@ionic/angular';
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+import { MockComponent } from 'ng-mocks';
+
+import { ItemComponent } from '../item';
+
+import { ItemGroupComponent } from './item-group.component';
+
+describe('ItemGroup', () => {
+  let spectator: SpectatorHost<ItemGroupComponent>;
+
+  const createHost = createHostFactory({
+    component: ItemGroupComponent,
+    declarations: [ItemGroupComponent, MockComponent(IonItem), MockComponent(ItemComponent)],
```

Comment:
I think it is sufficient to just mock `ItemComponent` 😄

```suggestion
    declarations: [ItemGroupComponent, MockComponent(ItemComponent)],
```

Reply:
I see, removed!